Dockerfile

```diff
@@ -21,9 +22,13 @@ RUN git clone ${POLY_GIT} ${POLY_DIR} && \
 RUN git clone ${OVEN_GIT} ${OVEN_DIR} && \
     cd ${OVEN_DIR} && \
     sha1sum worker.sml > cakeml-token && \
-    uname -norm > name && \
     polyc worker.sml -o worker
 
+RUN useradd -u 1050 -m oven && chown -R oven:oven /oven
+RUN dnf install -y psmisc diffutils
+USER oven
```

Comment:
There needs to be a chown somewhere because oven doesn't have write permissions on /.  We could avoid -R sure, but if I made any change to this file I wouldn't be committing exactly what I tested.

Reply:
Sure. Again, there is nothing wrong with the file as it is now I was just giving a review as requested by @xrchz. If you think it is too much of a hassle to change things or you do not agree with my comments, that is fine by me.

Reply:
I think we could also use `useradd -d /oven`.  Anyway, I was hoping to avoid another pass but since we need one anyway I'll do this when build 1357 finishes.

Reply:
And thank you for looking at this!  Sorry if i'm not conveying that well.